lib/tasks/foreman_vmware_checks_tasks.rake

```diff
@@ -27,23 +27,20 @@ namespace :test do
 end
 
 namespace :foreman_wreckingball do
-  task :rubocop do
-    begin
-      require 'rubocop/rake_task'
-      RuboCop::RakeTask.new(:rubocop_foreman_wreckingball) do |task|
-        task.patterns = ["#{ForemanWreckingball::Engine.root}/app/**/*.rb",
-                         "#{ForemanWreckingball::Engine.root}/lib/**/*.rb",
-                         "#{ForemanWreckingball::Engine.root}/test/**/*.rb"]
-      end
-    rescue StandardError
-      puts 'Rubocop not loaded.'
-    end
-
-    Rake::Task['rubocop_foreman_wreckingball'].invoke
+  require 'rubocop/rake_task'
```

Comment:
Wouldn't this overlap with the one in `Rakefile`? We used to need this for Jenkins but in GHA we test in a different way. I'd drop this part.